pkgs/top-level/all-packages.nix

```
@@ -24880,6 +24882,11 @@ with pkgs;
 
   https-dns-proxy = callPackage ../servers/dns/https-dns-proxy { };
 
+  hugegraph-unwrapped = callPackage ../servers/nosql/hugegraph/unwrapped.nix { };
+  hugegraph = callPackage ../servers/nosql/hugegraph {
+    inherit hugegraph-unwrapped;
+  };
```

```suggestion
  hugegraph = callPackage ../servers/nosql/hugegraph { };
```
callPackage should add that by itself